### ISSUE TITLE
EventLog logging configuration clarification

### ISSUE THREAD

**Issue by philip-reed** [dotnet/AspNetCore.Docs#18997]
In the section on windows event log, there is a sentence that reads:

> If EventLog log settings are specified, they default to LogLevel.Warning.

Should this be:
> If EventLog log settings are **not** specified, they default to LogLevel.Warning.

I am having an issue where the first statement is occurring.
I am specifying the LogLevel as Information, but it is still defaulting to Warning as per the documentation, which seems odd.

```
{
  "Logging": {
    "LogLevel": {
      "Default": "Debug"
    }
  },
  "EventLog": {
    "LogLevel": {
      "Default": "Information"
    }
  }
}
```

HostBuilder looks like this if it matters:
```
public static IHostBuilder CreateWebHostBuilder(string[] args) =>
            Host.CreateDefaultBuilder(args)
                .ConfigureWebHostDefaults(webBuilder =>
                {
                    webBuilder.UseStartup<Startup>()                       
                        .ConfigureLogging((hostingContext, logging) =>
                        {
                            logging.AddEventLog(options =>
                            {
                                options.SourceName = "ExampleApp";
                            });
                        })
                        .CaptureStartupErrors(true);
                });
```
Using core 3.1
Microsoft.Extensions.Logging.EventLog 3.1.5

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* Version Independent ID: 726e3bf1-f367-d733-8933-bccc04da0e16
* Content: [Logging in .NET Core and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#bilp)
* Content Source: [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/logging/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by serpent5** [comment]
> Should this be:
> 
> > If EventLog log settings are **not** specified, they default to LogLevel.Warning.

Yep, I think it should be _aren't_.

> I am specifying the LogLevel as Information...

This isn't what you're doing, because you have `EventLog` in the wrong section. It should be this:

```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Debug"
    },
    "EventLog": {
      "LogLevel": {
        "Default": "Information"
      }
    }
  }
}
```

Note that `EventLog` is a _child_ of `Logging`; not a _sibling_.

**Comment by philip-reed** [comment]
@serpent5 Thank you for the correction on my config.  That is working now.
What a coincidence that my mistake aligned with the typo!

**Comment by guardrex** [comment]
@philip-reed, do you want to put the PR in to use "aren't," or do want one of us to do it?

**Comment by philip-reed** [comment]
I've not contributed before so would like the opportunity.

Leave it with me to figure out.

**Comment by guardrex** [comment]
Sure thing ... use the "Edit" (pencil) on the topic and follow the prompts to submit the change.

Be sure to search **_the whole topic for multiple lines_** that might be wrong. We hold multiple versions of a topic in the same file. This sentence might appear two or more times. All instances that you find should be updated.

When you open the issue, put ...

```
Fixes #18997
```

... in the opening comment so that this issue will automatically close when the PR merges.

**Comment by philip-reed** [comment]
How's that?

Did it on my phone which was pretty cool.
Wasn't sure if should have updated the config example but figured if I got
it wrong then others might.
